README.md

```diff
@@ -108,11 +108,11 @@ For R users who are familiar with the `git` command line, gert should be mostly
 
 ### Automatic authentication
 
-To authenticate with a remote in git2r, you need to manually pass your crentials in every call to e.g. `git2r::clone()`, which is not ideal.
+To authenticate with a remote in git2r, you may need to manually pass your crentials in every call to e.g. `git2r::clone()`, which is not ideal.
```

Comment:
In theory, git2r can discover credentials. That probably only applies to ssh and, even then, as we know, often did not work. But I still think "may" is the right word here.